module_utils/kafka_lib_commons.py

```diff
@@ -236,7 +236,7 @@ def get_manager_from_params(params):
 
     api_version = tuple(
         int(p) for p in params['api_version'].strip(".").split(".")
-    )
+    ) if 'api_version' in params and params['api_version'] else None
```

Comment:
I think it can be simplified with
```suggestion
    ) if params.get('api_version') else None
```

Reply:
Done